unresolved-package-handler/base/deploymentconfig.yaml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  annotations:
+      thoth-station.ninja/template-version: 0.1.0
+  labels:
+    app: thoth
+    component: unresolved-package-handler-consumer
+  name: unresolved-package-handler-consumer
+spec:
+  replicas: 1
+  selector:
+    service: unresolved-package-handler-consumer
+  template:
+    metadata:
+      labels:
+        service: unresolved-package-handler-consumer
+    spec:
+      containers:
+        - image: unresolved-package-handler
+          name: unresolved-package-handler-consumer
+          env:
+            - name: SUBCOMMAND
+              value: "consumer"
+            - name: DEBUG_LEVEL
+              value: "0"
+            - name: APP_SCRIPT
+              value: "app.sh"
+            - name: KAFKA_BOOTSTRAP_SERVERS
+              valueFrom:
+                configMapKeyRef:
```

Comment:
the configmaps have been restructured, see https://github.com/thoth-station/thoth-application/blob/master/core/base/configmaps.yaml#L24 https://github.com/thoth-station/thoth-application/blob/master/core/overlays/test/configmaps.yaml#L43